README.md

```diff
@@ -4,21 +4,35 @@
 
 This is a generic skeleton project that can be used to quickly get a
 new [cisagov](https://github.com/cisagov) GitHub
-[Packer](https://packer.io) project started.  This skeleton project
+[Packer](https://packer.io) project started. This skeleton project
```

Comment:
Oh boy, here we go......  🤯

Reply:
:rage:

Reply:
![betty_white_dabbing](https://user-images.githubusercontent.com/50747025/81223912-d6de4e80-8fb4-11ea-8a9a-4138280f3d04.gif)